### PR TITLE
tui: high-contrast theme

### PR DESCRIPTION
**Pull request by mdhorn**
Fixes Issue: #217

Changes proposed in this pull request:
- Black and white theme to improve contrast

### CODE REVIEW

**Comment by jwakre** [comment]
When installing, I found a warning with a red background. Is this unexpected behavior for the high-contrast mode?

**Comment by mdhorn** [comment]
> When installing, I found a warning with a red background. Is this unexpected behavior for the high-contrast mode?

Probably not. It looks like for the warning/error we have Red (and bold white) hardcoded in a couple of places. I'll have to refactor this. Goog catch.

**Comment by mdhorn** [comment]
@jwakre The refactoring is completed and addressed the previous concerns.
All hard-coded colors are removed from the code and completely controlled by the themes.
Ready for review.

File: themes/clr-installer.theme [hunk 1]
```diff
@@ -101,6 +111,8 @@ TableView=─│┼▼▲
 ManualPartition.Back = blue
 ErrorLabel.Back = blue
 ErrorLabel.Text = red bold
+WarningLabel.Back = red
+WarningLabel.Text = white
```
Comment (karthikprabhuvinod) [inline themes/clr-installer.theme]: Somehow these didn't show up in red background and white text. 
Am i missing something?
warning

Comment (jwakre) [reply themes/clr-installer.theme]: This screenshot is from the high-contrast TUI? Your comment is on the clr-installer.theme file. There is a high-contrast.theme file below.

**Comment by mdhorn** [comment]
> Also, the focus in this theme, means an underline. any Specific reason we choose this? Why not over bar over the menu option?

The text-based UI doesn't allow boxes around the buttons, so we end up using light background and dark text to create a button box. In the color theme, clr-installer.theme, we can use different colors to differentiate between button boxes and the currently highlight/active text. In the high contrast/b&w, theme high-contrast.theme, we still need the light background and dark text to create the button boxes, so we instead use underline to identify the currently active text area.

File: themes/clr-installer.theme (hunk 1 above)
Comment (mdhorn) [reply themes/clr-installer.theme]: > Somehow these didn't show up in red background and white text.
> Am i missing something?
> warning

The high contrast is using the black and white only colors in the `high-contrast.theme`.

**Comment by karthikprabhuvinod** [comment]
LGTM

**Pull request merged**